packages/functional-tests/tests/settings/recoveryKey.spec.ts

```diff
@@ -154,3 +185,19 @@ test.describe('severity-1 #smoke', () => {
     });
   });
 });
+
+async function signInAccount(
+  target: BaseTarget,
+  page: Page,
+  login: LoginPage,
+  accountManager: AccountManager
+): Promise<Credentials> {
+  const credentials = await accountManager.signUp();
+  await page.goto(target.contentServerUrl);
+  await login.fillOutEmailFirstSignIn(credentials.email, credentials.password);
+
+  //Verify logged in on Settings page
+  expect(await login.isUserLoggedIn()).toBe(true);
+
+  return credentials;
+}
```

Comment:
I don't think helper/utility is technically right. I think ideally some or all of this should be in a POM, but the structure of POMS atm is in flux (see [ADR](https://github.com/mozilla/fxa/pull/16757#pullrequestreview-2028554920)), so I've opted to keep it repeating for now... open to suggestions

Reply:
Repeating is fine too. Or file a ticket to audit it in the future when/if it is problematic (ie, needs to be updated in 3+ places). Or this can easily be ignored if the helpers are subtly different for different pages, I only grepped for function names, I never did get my <kbd>grep -A10 ... | sort | uniq -c</kbd> hack working to see if it was the _exact_ same function or not.

Reply:
They are probably 90-95% the same I would guess, but yes my instinct is to follow-up.